Makefile

```diff
@@ -24,7 +24,7 @@ TESTBIN := build/bin/${BIN}-${GOOS}-${GOARCH}
 all: build
 
 prebuild:
-	go get -u github.com/gobuffalo/packr/packr
+	go install github.com/gobuffalo/packr/packr
```

Comment:
It was very annoying to pull latest master, run `make clean install` and be left with modified go.mod and go.sum files, signaling non-reproducible build. Part of mystery was that CI for gsctl already ensures `go mod tidy` is run and fails the build if either `go.mod` or `go.sum` were untidy. Running `go mod tidy` locally would clean things up again, so the difference in versions specified and versions used in the build wasn't due to difference in `go` version used in CI build and locally.

Analysis uncovered root cause to be `-u` switch in `go get` command, it would update packr dependency and its transitive dependencies, so not exactly respecting packr version specified in `go.mod` + `go.mod` files.

To fix that issue and to make it more obvious that `packr` binary is supposed to be installed, `go get -u` is replaced with `go install`.

Also, `tools.go` was added to track `packr` tool dependency explicitly using canonical Go modules approach (see [related official docs](https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module)).